game_view.rb

```diff
@@ -1,9 +1,12 @@
 class GameView
 
-  attr_reader :guessed_letters
+  attr_reader :guessed_letters, :body_array
```

Comment:
Also try not to name things based off of how they're implemented (especially when they're being accessed directly like this) you want to be printing `current_hangman_body` not `string_array`. It makes the code a lot more readable